### ISSUE TITLE
FPS Cap not working

### ISSUE THREAD

**Issue by Kralem** [ValveSoftware/Dota2-Gameplay#4928]
### Description

Dota 2 is ignoring my video settings in-game. I have my fps cap set to 140, but i am getting frame rates of over 200 in-game.
Not neccessarily world-ending, but it is bothersome because my gpu becomes louder because of the increased load.

**Comment by Reinoldo** [comment]
Same here. My processor is weak and it can't be overload so I set the fps cap to 90, now every 2 min the fps drops to 40 and I lose my mouse pointer in game.

**Comment by K4non82** [comment]
go in vulkan mode and hope this valve fix.. but i dont think theu can only broken the game in all patch,, maybe they can be tested before release

**Comment by Akz77** [comment]
same issue.

**Comment by tontanck** [comment]
same here. I also tried to reinstall both the game and NVIDIA Graphic drivers, not of them worked. other games are working just fine so I'm pretty sure it must be Valve's bug. please Let me know if there is any update about it.

**Comment by tontanck** [comment]
### Mine is now fixed after a Dota update !!!!!

**Comment by K4non82** [comment]
Yes me too.. of cant reach max FPS, likes me.. i do 158 FPS on 165, male off Nvidia reflex
--
Inviato da Libero Mail per Android martedì, 08 novembre 2022, 01:51PM +01:00 da tontanck  ***@***.*** :
